src/batou_ext/cron.py

```diff
@@ -59,6 +59,10 @@ def configure(self):
         self.lock_file = self.map(self.expand(".{{component.tag}}.lock"))
         self.stamp_file = self.map(self.expand(".{{component.tag}}.stamp"))
 
+        # Ensure, we do have the stamp file there so the sensu-check is
+        # getting red if next runs are not successful
+        self += batou.lib.file.File(self.stamp_file)
```

Comment:
This most likely fails. If you want an empty file, you need to pass `content=""`.

Reply:
Should be better now